exchange/exchange.go

```diff
@@ -130,12 +130,8 @@ func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidReque
 				priceGranularity:  requestExt.Prebid.Targeting.PriceGranularity,
 				includeWinners:    requestExt.Prebid.Targeting.IncludeWinners,
 				includeBidderKeys: requestExt.Prebid.Targeting.IncludeBidderKeys,
-			}
-			if shouldCacheBids {
-				targData.includeCacheBids = true
-			}
-			if shouldCacheVAST {
-				targData.includeCacheVast = true
+				includeCacheBids:  shouldCacheBids,
```

Comment:
Unrelated to PR title. A simple refactor opportunity I stumbled upon.

Reply:
Isn't the previous code cleaner? Why do you think the `if` statements are preferable? I believe they add unnecessary identation and statements to the code and make it a bit less readable. Plus, they add a couplr of conditionals to the code and according to SonarCloud, it is preferable to have at most 15 of those.